Bots/DotNet/Skills/Composer/ComposerSkillBotDotNet/ComposerSkillBotDotNet/README.md

```diff
@@ -0,0 +1,27 @@
+# Welcome to your new bot

```

Comment:
We can probably remove this file (and the other readme.md form the other composer projects too)